.github/workflows/parachain.yml

```diff
@@ -84,6 +91,7 @@ jobs:
           --workspace
           --features runtime-benchmarks
           --exclude snowbridge-ethereum-beacon-client
+          --exclude snowbridge-runtime-tests
```

Comment:
Do we also now need to run `cargo test --exclude snowbridge-runtime-tests` on the command line too?